prereise/gather/griddata/hifld/const.py

```diff
@@ -438,6 +511,18 @@
     "west": -125,
 }
```

Comment:
Should we then delete these lines?

Reply:
The bounds? We still use them within `get_hifld_electric_power_transmission_lines`

Reply:
Sorry,, I was referring to `nercregion2interconnect`. The comment says: Usage of this is deprecated, since these data seem noisier than Balancing Authorities

Reply:
We do still use `nercregion2interconnect` if `balancingauthority2interconnect` doesn't work for all generators (e.g. the `WACM`, `WALC`, and `WAUW` BAs). I think when I wrote the comment I thought we could use `balancingauthority2interconnect` alone, so I should probably update the comment to make it a little clearer that this is still used, but only as a fallback.

Reply:
@danielolsen That's consistent with my understanding. The fallback option got eventually used in some corner cases, which we failed to eliminate.